lib/Timber.php

```diff
@@ -528,9 +528,19 @@ public static function compile( $filenames, $data = array(), $expires = false, $
 				);
 			}
 
-			$output = $loader->render($file, $data, $expires, $cache_mode);
+			$output = $loader->render( $file, $data, $expires, $cache_mode );
 		}
 
+		/**
+		 * Replaces the depraceted `timber_compile_result`.
```

Comment:
The DocBlock needs to start with "Filters" so that it will be picked up for the [Hook Documentation](https://github.com/timber/timber/blob/master/CONTRIBUTING.md#documenting-hooks).

```suggestion
		 * Filters the compiled result before it is returned in `Timber::compile()`.
		 *
```